sopel/builtins/tld.py

```diff
@@ -292,9 +292,9 @@ def _update_tld_data(bot, which, force=False):
 
 
 @plugin.interval(60 * 60)
-def update_caches(bot, force=False):
-    _update_tld_data(bot, 'list', force)
-    _update_tld_data(bot, 'data', force)
+def update_caches(bot):
+    _update_tld_data(bot, 'list', False)
+    _update_tld_data(bot, 'data', False)
```

Comment:
Could be problematic if doing this is no longer supported, certainly if targeting this for a .1 release. In builtins and third-party plugins, having optional parameters to modify the default behavior of a `@plugin.interval` job isn't _terribly_ uncommon.

Reply:
Yeah I get you. I really don't like it, but I think I have a workaround, i.e. accept `*arg` and `**kwargs` in both PluginCallable and PluginJob's call methods.

Reply:
Fixed!